### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by smitpatel**
@AndriySvyryd - Here it is. I did it half way through test projects. I couldn't go past EFCore.Specs.Tests.

Resharper just crashes VS all the time. It also caused some compilation error which I had to revert.
Most of the changes are teeny-tiny space changes which are providing marginal values.
If anyone wants to run code cleanup, its fine. I don't want to try it any further.

### CODE REVIEW

File: samples/OracleProvider/src/OracleProvider/Internal/OracleLoggerExtensions.cs [hunk 1]
```diff
@@ -140,11 +139,11 @@ private static string ByteIdentityColumnWarning(EventDefinitionBase definition,
...
-            [NotNull] this IDiagnosticsLogger<DbLoggerCategory.Scaffolding> diagnostics,
-            [NotNull] string foreignKeyName,
-            [NotNull] string tableName,
-            [NotNull] string principalTableName,
-            [NotNull] string onDeleteAction)
+                [NotNull] this IDiagnosticsLogger<DbLoggerCategory.Scaffolding> diagnostics,
+                [NotNull] string foreignKeyName,
+                [NotNull] string tableName,
+                [NotNull] string principalTableName,
+                [NotNull] string onDeleteAction)
```
Comment (ralmsdeveloper) [inline samples/OracleProvider/src/OracleProvider/Internal/OracleLoggerExtensions.cs]: @smitpatel Will you use 8 spaces?

Comment (ralmsdeveloper) [reply samples/OracleProvider/src/OracleProvider/Internal/OracleLoggerExtensions.cs]: @smitpatel , sorry for the silly question, looking better, I saw that in Expression Body Definitions you use 8 spaces!

Have patience with me, little by little I get there!

I believe asking is the best way to improve!

Comment (AndriySvyryd) [reply samples/OracleProvider/src/OracleProvider/Internal/OracleLoggerExtensions.cs]: This looks like a R# bug. Do you have the newest version?

Comment (ralmsdeveloper) [reply samples/OracleProvider/src/OracleProvider/Internal/OracleLoggerExtensions.cs]: Thanks @AndriySvyryd, 
I was already getting confused, until I started looking at all code. 
there are many places that are like this!

An example:
https://github.com/aspnet/EntityFrameworkCore/blob/dev/src/EFCore.SqlServer/Internal/SqlServerLoggerExtensions.cs#L141

Comment (smitpatel) [reply samples/OracleProvider/src/OracleProvider/Internal/OracleLoggerExtensions.cs]: @AndriySvyryd - I have 2017.2.2 i think. Probably not the latest.

**Comment by smitpatel** [comment]
I am merging this PR. Someone else in team can later check/correct/re-run code cleanup using R# to get things fixed.

**Pull request merged**